react-assissans/src/components/navbar.js

```diff
@@ -0,0 +1,71 @@
+import { Navbar, Nav, Spinner } from "react-bootstrap";
+import React, { useState } from "react";
+const moviesUrl =
+  "https://api.themoviedb.org/3/search/movie/?api_key=1d54e327869a62aba4dc1b58c2b30233";
+
+function navBar(props) {
+  return (
+    <>
+      <Navbar bg="dark" variant="dark">
+        <Navbar.Brand href="#">Assassins Movies</Navbar.Brand>
+        <Nav>
+          <Nav.Link href="#home">Home</Nav.Link>
+        </Nav>
+        <SearchBox onHandle={props.handleQuery} onSubmit={props.handleMovies} />
+      </Navbar>
+    </>
+  );
+}
+
+function SearchBox(props) {
+  const [value, setValue] = useState("");
+  let [loading, setLoading] = useState(false);
+
+  const handleChange = (event) => {
+    setLoading(true);
+    setValue(event.target.value);
+    props.onHandle(event.target.value);
+    moviesData(value).then((moviesArr) => {
+      setLoading(false);
+    });
+  };
+  const handleSubmit = (event) => {
+    setLoading(true);
```

Comment:
onSubmit the spinner should be hidden so do you think the right value here is true?